lib/countwords.rb

```diff
@@ -0,0 +1,43 @@
+class CountWords
+
+  attr_accessor :word_count
+
+  def count_words_from_file(file)
+    count_file = File.open(file, "r")
+    count_file.each_line do |line|
+      next if line.empty?
+      @@word_count = Hash.new
+      word = line.split
+      word.each do |wrd|
+        next if wrd !~ /(\w+)/
+        wrd = wrd[/(\w+)/]
```

Comment:
Splitting the line, iterating, and making sure each word matches `\w+`, can be simplified using [`String#scan`](http://ruby-doc.org/core-2.2.2/String.html#method-i-scan).